src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,62 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+type Tab = {
+  id: string,
+  title: string,
+  content: string
+};
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (a: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const selectHandler = (
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+    isSelected: Tab,
```

Comment:
the variable `is...`  says that there will be a boolean value, and there will be a number, looks confusing. Rename the variable to a more relevant name. Read about [boolean naming](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig) for better understanding